src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1425,15 +1425,15 @@ export function registerTerminalActions() {
 			const terminalService = accessor.get(ITerminalService);
 			const workspaceContextService = accessor.get(IWorkspaceContextService);
 			const options = convertOptionsOrProfileToOptions(optionsOrProfile);
-			const activeInstance = terminalService.getInstanceHost(options?.target).activeInstance;
+			const activeInstance = terminalService.getInstanceHost(options?.location === TerminalLocation.Editor ? TerminalLocation.Editor : TerminalLocation.Panel).activeInstance;
```

Comment:
`getInstanceHost` accepts undefined here for default, shouldn't this be this?

```ts
const activeInstance = terminalService.getInstanceHost(options?.location).activeInstance;
```

Reply:
options.location can be `TerminalLocation | TerminalEditorLocation | { parentTerminal: ITerminalInstance };`